niworkflows/anat/ants.py

```diff
@@ -549,10 +587,12 @@ def init_atropos_wf(
 
     # massage outputs
     pad_segm = pe.Node(
-        ImageMath(operation="PadImage", op2="%d" % padding), name="02_pad_segm"
+        ImageMath(operation="PadImage", op2=f"{padding}", copy_header=False),
```

Comment:
If ANTs is potentially screwing up the affines, should we just implement padding ourselves?

Reply:
Yes, yes, yes :)

- #568